frontend/src/js/services/stringToTemplate.js

```diff
@@ -0,0 +1,24 @@
+
+angular
+    .module('evalai')
+    .service('stringToTemplate', stringToTemplate);
+
+function stringToTemplate() {
+
+    this.convert = function (endpoint, params) {
+
+        var key = Object.keys(params)[0];
+        var value = params[key];
+        this.convertString = function(literal, params) {
+            return new Function(params, "return `"+literal+"`;");
+        };
+
+        let template = this.convertString(endpoint, key);
```

Comment:
@aayusharora don't use let here.

Reply:
What should I use here @aka-jain  and why, it will be helpful if you provide some description.

Reply:
Use var only, I think let works best with ES6. And we are not using any transcompiler to compile the ES6 code.

Reply:
Hi @aka-jain It will perfect, as the templating syntax of ES6 is working perfect `` as well.

Reply:
Not every browser support this. Give it a read. Older version of chrome and safari are still at risk. So, it would be better if use var here. @aayusharora

Reply:
@spyshiv Please take this up so that we can merge this PR.

Reply:
Hi @aka-jain I don't find any link in your previous comment.
But please check the snapshot.
![screen shot 2018-01-07 at 12 36 03 am](https://user-images.githubusercontent.com/12194719/34643007-dab3462a-f342-11e7-9517-fcfbed6a1198.png)

Reply:
For ``  ![screen shot 2018-01-07 at 02 10 00](https://user-images.githubusercontent.com/11537940/34643768-1a902440-f350-11e7-97a3-e17007eb51d3.png)

and check for other versions too https://caniuse.com/#feat=let. Also for the consistency I suggest not to use ES6 here. Will do it if we migrate to Angular 2+
@aayusharora

Reply:
@aayusharora it's getting too long for no reason. Let me know what we can do for this.

Reply:
Hi @deshraj  Can you look into this, as we have decided at the start of this PR, 
it was suggested by you to go with template literals as without that we will not be able to produce the solution which was desired.

I am unable to follow the above comments.
Thanks

Reply:
@spyshiv I think you should take this up. 
Also, @aayusharora if you are doing something then please discuss it with us too so that we can figure out the problem in advance.